docs/sources/access/index.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Access or install Explore Profiles.
+keywords:
+  - Install
+  - Configure
+  - Explore Profiles
+title: Access or install Explore Profiles
+menuTitle: Access or install
+weight: 150
+---
+
+# Access or install Explore Profiles
+
+Explore Profiles is a native Grafana application designed to integrate seamlessly with [Pyroscope](https://github.com/grafana/pyroscope), the open-source continuous profiling platform, providing a smooth, query-less experience for browsing and analyzing profiling data.
+
+You can use Explore Profiles in Grafana Cloud or with Grafana open source stand-alone.
+
+## Before you begin
+
+To use Explore Profiles with Grafana Cloud, you need:
+
+* A Grafana Cloud account
+* A Grafana stack in Grafana Cloud with a configured Hosted profiles or Pyroscope data source
```

Comment:
I'm not sure if the "Hosted" needs to be in caps here.

Reply:
You can use either the Hosted profiles data source or the Pyroscope data source. I capitalized it to match the Grafana UI. Do you find it confusing?  
![image](https://github.com/user-attachments/assets/7dc1b18d-f8f3-4e1c-9a94-25c9051d4b0c)

Reply:
For example: 
![image](https://github.com/user-attachments/assets/0d98f13f-2016-4e44-8073-017028c33961)

Reply:
Only because I'm not looking at the UI. I think that if it's a known data source, it shouldn't be confusing for users.